server.js

```diff
@@ -1,23 +1,23 @@
-const express = require('express');
-const exphbs = require('express-handlebars');
+const express = require("express");
+const exphbs = require("express-handlebars");
 
 const app = express();
 const PORT = process.env.PORT || 8080;
-const home = require('./router/home');
-const about = require('./router/about');
-const contact = require('./router/contact');
+const home = require("./routes/home/get");
+const about = require("./routes/about/get");
+const contact = require("./routes/contact/get");
```

Comment:
Instead of going directly to the routes, figure out why the `./routes/*` index.js file isn't working.